tests/profiles/generate_profiles.py

```
@@ -0,0 +1,120 @@
+    secrets_b64 = os.environ.get(secrets_json_env, "").strip()
+
+    # ── Build template context ──────────────────────────────────────────
+    context: dict[str, object] = {"schema_name": schema_name}
+
+    if secrets_b64:
+        try:
+            decoded: dict = json.loads(base64.b64decode(secrets_b64))
+        except (ValueError, json.JSONDecodeError) as e:
+            raise click.ClickException(
+                f"Failed to decode ${secrets_json_env}: {e}"
+            ) from e
```

Ruff (0.15.2)

[warning] Avoid specifying long messages outside the exception class

(TRY003)